bin/dev_command/setup

```diff
@@ -112,7 +135,7 @@ setup() {
     mysql;
 
     feature RabbitMQ rabbitmq;
-    feature "Percona 8" mysql8;
+    feature "Percona 8" mysql8; mysql 8;
```

Comment:
I thinks this is not right

Reply:
This is intentional.
I adjusted the mysql function to be able to provision multiple hosts. Since the name of the service differs from the name of the config I have to know the actual version here in order to also be able to check before checking if the volume needs to be created and running the `dev setup-mysql db8`.